src/main/java/com/tarento/assetManagement/service/HelloWorldService.java

```diff
@@ -0,0 +1,16 @@
+package com.tarento.assetManagement.service;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HelloWorldService {
```

Comment:
why HelloWorld service required.

Reply:
These including test cases were given inbuilt, i thought once the coding is done i will be asked to write test cases so i added them to git, if not needed will remove them.